spec/wwtd_spec.rb

```diff
@@ -150,17 +151,20 @@ def write_default_rakefile
       wwtd("").should include "RUBY: 2.0.0"
     end
 
-    it "runs with given rvm version" do
-      other = (RUBY_VERSION == "1.9.3" ? "2.0.0" : "1.9.3")
-      write ".travis.yml", "rvm: #{other}"
-      write "Rakefile", "task(:default) { puts %Q{RUBY: \#{RUBY_VERSION}} }"
-      wwtd("").should include "RUBY: #{other}"
+    def ruby_list
+      %w(2.0.0 2.1.5 2.2.1)
```

Comment:
that will just break again, maybe try to read the list of currently installed rubies or just leave it as it was